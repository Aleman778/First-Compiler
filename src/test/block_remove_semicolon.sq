fn foo() -> i32 {
    10
}


fn main() {
    let x: i32 = {
        
        foo(); //~ error: mismatched types, remove semicolon
    };
}
