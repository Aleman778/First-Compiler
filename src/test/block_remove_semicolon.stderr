error: mismatched types
  --> $DIR/block_remove_semicolon.sq:7:18
   |
LL |       let x: i32 = {
   |  __________________^
LL | |
LL | |         foo(); //~ error: mismatched types, remove semicolon
LL | |     };
   | |_____^ expected `i32`, found `()`